utils/utils.hre.ts

```diff
@@ -6,6 +6,7 @@ export async function deployNewProxy(name: string, args: (number | string)[]): P
 
   const proxy = await upgrades.deployProxy(await getContractFactory(name, {}), args, {
     kind: "uups",
+    unsafeAllow: ["delegatecall"],
```

Comment:
I wasn't able to deploy without this, so I left it in. @nicholaspai is there somewhere else this should be specified or is this correct?

Reply:
FWIW I hit the same issue on zkSync. It seems like we are importing AddressUpgradeable.sol via Initialisable.sol. @nicholaspai found that the import was recently removed, so it's no longer there on master. _Hopefully_ we can remove this suppression after updating the dependency.

https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/commit/15bc82434a56adc1113e70f7b2ad7f2848a641ee

Reply:
Yeah it depends on a new OZ NPM package unfortunately